pylint_odoo/checkers/modules_odoo.py

```diff
@@ -128,6 +128,11 @@
         'xml-attribute-translatable',
         settings.DESC_DFLT
     ),
+    'W%d39' % settings.BASE_OMODULE_ID: (
+        '%s Deprecated <data> xml node indide <odoo> xml node',
```

Comment:
And append the following message: Use `<odoo>` instead of `<odoo><data>`  or use `<odoo noupdate="1">` instead of `<odoo><data noupdate="1">`